docs/installation/README.md

```diff
@@ -0,0 +1,28 @@
+# Installation
+If you want to install La Suite Meet you've come to the right place.
+Here are a bunch of resources to help you install the project.
+
+## Kubernetes
+La Suite Meet maintainers use only the Kubernetes deployment method in production, so advanced support is available exclusively for this setup. Please follow the instructions provided [here](/docs/installation/kubernetes.md).
+
+## Docker Compose
+We understand that not everyone has a Kubernetes cluster available.
+We also provide [Docker images](https://hub.docker.com/u/lasuite?page=1&search=meet) that can be deployed using Compose.
+
+> [!WARNING]  
+> Under construction: A PR is in progress to support deploying La Suite Meet via Docker Compose.
+
+## Other ways to install La Suite Meet
+Community members have contributed alternative ways to install La Suite Meet 🙏. While maintainers may not provide direct support, we help keep these instructions up to date, and you can reach out to contributors or the community for assistance.
+
+Here is the list of other methods in alphabetical order:
+- Nix: [Packages](https://search.nixos.org/packages?channel=unstable&show=lasuite-meet&query=lasuite-meet), ⚠️ unstable
+- Yunohost: [Packages](https://github.com/YunoHost-Apps/meet_ynh), ⚠️ under construction
```

Comment:
maybe worth to mention that it is meant for small instances (Yunohost is not meant to scale)